docs/node/migrations/offline-migrations/network-snapshots.mdx

```diff
@@ -0,0 +1,89 @@
+---
+sidebar_position: 1
+sidebar_label: "Network Snapshots"
+id: network-snapshots
+title: Network Snapshots
+description: Network Snapshots
+slug: /node/migrations/offline-migrations/network-snapshots
+---
+
+Kwild supports network snapshots, which captures the state of the application at a specific point in time. These network snapshots can be used to initialize a new network with the same state as the existing network at the time of the snapshot, which makes them useful for network migrations and upgrades. These snapshots include data related to user-deployed schemas, accounts, processed votes on resolutions and more. 
+
+## Create Snapshot
+
+Network snapshots can be generated using the [kwil-admin snapshot](/docs/ref/kwil-admin/snapshot/create) tool. This tool connects directly to the database to download its state and does not require the `kwild` process to be running. 
+
+```bash
+kwil-admin snapshot create --snapdir /path/to/snapdir
+```
+
+By default, the `snapshot create` command connects to the postgres instance runnning at `localhost:5432` with the default user `postgres` with `trust` authentication mode. To connect to a database running on a different host/port/user/password, use the following command:
+
+```bash
+kwil-admin snapshot create --snapdir /path/to/snapdir --dbname <dbname> --host <dbhost> --port <dbport> --user <dbuser> --password <dbpass>
+```
+
+The snapshot tool generates a snapshot file and a genesis file required to start the new network as shown below.
+
+```bash
+# Running the above command will generate the snapshot file and the genesis.json file in the snapshot directory.
+ls /path/to/snapdir
+genesis.json            kwildb-snapshot.sql.gz
+```
```

Comment:
How do you feel about reorganizing the above section as follows?

<pre>
Network snapshots can be generated using the [kwil-admin snapshot create](/docs/ref/kwil-admin/snapshot/create) command.
This tool connects directly to the database to download its state and does not require the `kwild` process to be running.

```sh
kwil-admin snapshot create --snapdir /path/to/snapdir
```

This generates a compressed snapshot file and a template genesis file required to start the new network as shown below.

```sh
$ ls /path/to/snapdir
genesis.json            kwildb-snapshot.sql.gz
```

:::tip
By default, the `snapshot create` command connects to the `postgres` instance running at `localhost:5432` with the default user `postgres` (without a password?). To modify the database connection settings, see the `kwil-admin` reference page.
:::

<pre>